Apps/Src/SendCarCAN.c

```diff
@@ -48,19 +50,27 @@ uint8_t get_SendCarCAN_Q_Space(void) {
 
 /**
  * @brief Wrapper to put new message in the CAN queue
- * @param message the CAN message to put in SendCarCAN Queue
 */
 void SendCarCAN_Put(CANDATA_t message){
-    OS_ERR err;
+    OS_ERR err = OS_ERR_NONE;
```

Comment:
```suggestion
    OS_ERR err;
```
I'm not sure which way is better, but we could delete this and put the assertOSError into the conditional (so it isn't asserted if nothing happens) to be more consistent with other parts of the codebase.